docs/software/firmware/csv_format.md

```diff
@@ -0,0 +1,64 @@
+# Format specification for the internal CVS format
+
+In addition, the will be a json file per CSV file containing some metadata. 
+The file needs to be specified later - still there is some data needed here - 
+so I will list it.
+
+## Metadata
+
+OBSVersion
+: v0.3.999
+
+FormatVersion
+: 2
+
+DatasPerMeasurement
+: 3 (fix as of now is `Tms<n>`, `Lus<n>` and `Rus<n>`)
+
+MaximumMeasurementsPerLine
+: 40 (currently fix)
+
+## CSV
+
+The CSV File must contain a header line with headline entries for each field, the 
+names must be the same as given here. The number of data entries per line
+can differ.
+
+NOTE: The order of the fields is different than in Version 1 of the CVS file. Ok??
+
+Headline  | Format | Range | Sample | Description |
+---       | --- | --- | --- | --- |
+`Date`    | TT.MM.YYYY | | 24.11.2020 | UTC, typically as received by the GPS module in that second. If there is no GPS module present, system time is used. If there was no reception of a time signal yet, this might be unix time (starting 1.1.1970) which can be at least used as offset between the csv lines.    
+`Time`      | HH.MM.SS | | 12:00:00 | UTC time, see also above
+`Latitude`  | Float |  | 9.123456 | Latitude as degrees
+`LatitudeDir` | Char | `N` /  `S` | N | Latitude direction as reported by the GPS module (GPGGA)
+`Longitude` | Float |  | 42.123456 | Longitude in degrees
+`LongitudeDir` | Char | `E` / `W` | W | Longitude direction as reported by the GPS module (GPGGA)
```

Comment:
N/S and W/E are already encoded as signs into Latitude/Longitude, so the information in LatitudeDir and LongitudeDir is redudant.
I propose the remove LatitudeDir and LongitudeDir.

Reply:
Top - I was not sure about this!